_haikus/diwali_lights.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Diwali Lights
+author: Avi Kathuria
+---
+
+Festival of Lights<br>
+Arrays of Diyas Burn Bright <br>
+Inner Space Glows <br>
```

Comment:
```suggestion
The Inner Space Glows <br>
```

Reply:
@ltagliaferri : thanks for update, please check now.

Checked syllable count as well:
<img width="776" alt="Screenshot 2022-10-30 at 11 34 35 PM" src="https://user-images.githubusercontent.com/19428377/198894183-1c7e225c-b4f6-44f9-945a-8422ef142347.png">